perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/CabinetManagerMethod.java

```diff
@@ -86,13 +87,33 @@ public PublicationSystem call(ApiCaller ac, Deserializer parms) throws PerunExce
 	/*#
 	 * Creates new Category for Publications with specified name and rank.
 	 *
-	 * @param category Category new Category object
+	 * Category object's parameters name and rank are optional. Other parameters are ignored.
```

Comment:
While it is not required on server side API, we always submit Category object with non empty name (limited to 128 chars) and non-empty rank value (double with single digit after decimal point). Please update the docs in this manner, we will later update server side checks.